pom.xml

```diff
@@ -282,9 +282,9 @@
           </jvmArgs>
           <javacArgs>
             <javacArg>-source</javacArg>
-            <javacArg>${java.version}</javacArg>
+            <javacArg>1.7</javacArg>
```

Comment:
Sorry, I got cornered into doing this for a reason that I didn't have paged in and meant to go back and try to reconstruct what that was.

I'm pretty sure it was related to my making the build use `zinc`; for some reason `1.8` was not acceptable in that setting, and I had to hard-code `1.7` here. I'll try to dig back and get the full state on it for posterity, but in the meantime I was going to give up on `zinc` in this PR anyway, since I didn't know how to get around the "`sbt-interface.jar` not found" problem on Jenkins discussed elsewhere on this PR (I believe due to `zinc` not being installed on Jenkins in some way), so I'll revert this bit momentarily. Thanks!